specification/introduction/index.md

```diff
@@ -41,11 +41,11 @@ To allow for such in-depth control over specific content types and formats, E-AR
 
 As of November 2018 these Content Information Type Specifications, created by the E-ARK project and enchanced by the DILCIS Board, have been verified for usage within the Common Specification for Information Packages:
 
-- E-ARK ERMS: Content Information Type Specification Electronic Records Management Systems. (<https://github.com/DILCISBoard/E-ARK-ERMS>) This Content Information Type Specification describes the use of the CSIP for the archiving of records exported from ERMS-type systems;
+- E-ARK Content Information Type Specification for ERMS: This Content Information Type Specification for Electronic Records Management Systems. (<https://github.com/DILCISBoard/E-ARK-ERMS>) This Content Information Type Specification describes the use of the CSIP for the archiving of records exported from ERMS-type systems;
 
-- E-ARK Geodata: This Content Information Type Specifications build upon the INSPIRE directive and describes the storing geospatial information (<https://github.com/DILCISBoard/E-ARK-Geodata>), within a CSIP compatible Information Package;
+- E-ARK Content Information Type Specification for Geospatial data: This Content Information Type Specification build upon the INSPIRE directive and describes the storing of geospatial information (<https://github.com/DILCISBoard/E-ARK-Geodata>), within a CSIP compatible Information Package;
```

Comment:
- E-ARK Content Information Type Specification for Geospatial data: This Content Information Type Specification builds upon the INSPIRE directive and describes the storage of geospatial information within a CSIP compatible Information Package (<https://github.com/DILCISBoard/E-ARK-Geodata>);